src/OrchardCore.Mvc.Web/OrchardCore.Mvc.Web.csproj

```diff
@@ -5,6 +5,8 @@
   <PropertyGroup>
     <TargetFrameworks>$(CommonTargetFrameworks)</TargetFrameworks>
     <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+    <Nullable>enable</Nullable>
```

Comment:
Are the PR related to `Orchard.Cms.Web` or something else?

Reply:
related to title :) `Orchard.Cms.Web with .NET 6`.

Reply:
It's confusing because the title mentioned Orchard.Cms.Web ;)

Reply:
Updated title :)